src/app/screens/SignUp/components/CustomInput/styles.module.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../../../scss/variables/colors';
```

Comment:
when you have something inside `scss/variables` you can import like `@import 'variables/colors';`